analyses/mwi/dryspells_trigger/exploration/mwi_agg_exploration.Rmd

```diff
@@ -0,0 +1,382 @@
+---
+title: "Comparison of MWI Observational Aggregation Methods"
+author: "Seth Caldwell"
+date: "15/12/2021"
+output: ioslides_presentation
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+
+library(tidyverse)
```

Comment:
I had to do `install.packages("Rcpp")`. Not sure if that should be defined in the proj file or so?

Reply:
No, would be if we were using something like `renv` but I think that's overkill, should be all good now!